hermione.js

```diff
@@ -61,21 +61,21 @@ function prepareImages(hermione, pluginConfig, reportBuilder) {
         let queue = Promise.resolve();
 
         hermione.on(hermione.events.TEST_PASS, (testResult) => {
-            queue = queue.then(() => saveTestImages(reportBuilder.format(testResult), reportPath));
+            queue = queue.then(() => saveTestImages(reportBuilder.format(testResult), reportPath)).catch(reject);
         });
 
         hermione.on(hermione.events.RETRY, (testResult) => {
-            queue = queue.then(() => failHandler(testResult));
+            queue = queue.then(() => failHandler(testResult)).catch(reject);
```

Comment:
```js
queue = queue.then(() => failHandler(testResult), reject);
```
?

Reply:
This won't catch error from this `failHandler`, only from previous one